Lib/test/test_enumerate.py

```diff
@@ -160,9 +160,9 @@ def __getitem__(self, i):
                 raise StopIteration
             def __len__(self):
                 return 5
-        for data in 'abc', range(5), tuple(enumerate('abc')), A(), range(1,17,5):
+        for data in ('abc', range(5), tuple(enumerate('abc')), A(),
+                    range(1,17,5), {i: i for i in range(20)}):
```

Comment:
Again, `dict.fromkeys('abcdefg')` might be better here than `{i: i for i in range(20)}`.